src/rprblender/properties/render.py

```diff
@@ -493,6 +499,11 @@ def init_rpr_context(self, rpr_context, is_final_engine=True, use_gl_interop=Fal
             if float(mac_vers_major) >= 14:
                 rpr_context.set_parameter(pyrpr.CONTEXT_METAL_PERFORMANCE_SHADER, 1)
 
+        # enable texture cache
+        if not os.path.isdir(self.texture_cache_dir):
+            os.mkdir(self.texture_cache_dir)
+        rpr_context.set_parameter(pyrpr.CONTEXT_TEXTURE_CACHE_PATH, self.texture_cache_dir)
```

Comment:
CONTEXT_TEXTURE_CACHE_PATH is available only for RPR 2, please add check isinstance(rpr_context, RPRContext2) or override RPRContext2.set_parameter() with ignoring CONTEXT_TEXTURE_CACHE_PATH option.